deploy/lib/ml_rest.rb

```diff
@@ -190,19 +158,19 @@ def install_extensions(path)
                 end
               end
             end
-
+          
           end
-
+          
           @logger.debug "extensionName: #{extensionName}"
           @logger.debug "headers: #{headers}"
           @logger.debug "params: #{params}"
-
-          url = "#{@rest_protocol}://#{@hostname}:#{@port}/v1/config/resources/#{extensionName}"
+          
+          url = "http://#{@hostname}:#{@port}/v1/config/resources/#{extensionName}"
           if (params.length > 0)
-            url << "?" << params.join("&")
+            url << "?name=#{extensionName}&" << params.join("&")
```

Comment:
Looking at [PUT /v1/config/resources](http://docs.marklogic.com/REST/PUT/v1/config/resources/[name]), I don't see a name parameter. Looks to me like the extensionName should be part of the URL, rather than a URL parameter. What am I missing?